modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/ScheduleContainer.tsx

```diff
@@ -80,10 +80,10 @@ export default function ScheduleContainer({
 					<ScheduleField
 						checkboxLabel={Liferay.Language.get('never-expire')}
 						customValidation={(value) => {
-							const currentDate = new Date().setSeconds(0, 0);
-							const dateTime = new Date(value).setSeconds(0, 0);
+							const currentDate = new Date();
+							const dateTime = new Date(value);
```

Comment:
Both dates have times, right? Why does only one have the suffix "time"?

Use `currentDateTime` and `dateTime` or `currentDate` and `date`.

Apply the same chosen pattern to the JSP